shared/subprocess/proc.go

```diff
@@ -0,0 +1,136 @@
+package subprocess
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
```

Comment:
Similar comment to other file, the `gopkg.in/yaml.v2` should be separated from the rest as an external import.